samples/proofs/src/App.tsx

```diff
@@ -0,0 +1,120 @@
+        // In a production scenario the challenge should not be hardcoded, in order to avoid accepting proofs created for other contexts.
+        const challenge = 'beefbeefbeefbeefbeefbeefbeefbeefbeefbeefbeefbeefbeefbeefbeefbeef';
+
+        connection
+            ?.requestVerifiablePresentation(challenge, statement)
+            .then((res) => setVerifiablePresentation(res))
+            .catch((e: Error) => setError(`Failed to get verifiable presentation: ${e.message}`))
+            .finally(() => setIsWaiting(false));
+    }, [connection]);
+
+    return (
+        <>
+            <Row className="mt-3 mb-3">
+                <Col>
+                    <WalletConnectorButton
+                        connectorType={BROWSER_WALLET}
+                        connectorName="Browser Wallet"
+                        connection={connection}
+                        {...props}
+                    />
+                </Col>
+                <Col>
+                    <WalletConnectorButton
+                        connectorType={WALLET_CONNECT}
+                        connectorName="WalletConnect"
+                        connection={connection}
+                        {...props}
+                    />
+                </Col>
+            </Row>
+            <Row className="mt-3 mb-3">
+                <Col>
+                    {activeConnectorError && <Alert variant="danger">Connector error: {activeConnectorError}.</Alert>}
+                    {!activeConnectorError && activeConnectorType && !activeConnector && <Spinner />}
+                    {connectError && <Alert variant="danger">Connection error: {connectError}.</Alert>}
+                    {activeConnector && !account && (
+                        <Button type="button" onClick={connect} disabled={isConnecting}>
+                            {isConnecting && 'Connecting...'}
+                            {!isConnecting && activeConnectorType === BROWSER_WALLET && 'Connect Browser Wallet'}
+                            {!isConnecting && activeConnectorType === WALLET_CONNECT && 'Connect Mobile Wallet'}
+                        </Button>
+                    )}
+                </Col>
+            </Row>
+            {account && (
+                <Row className="mt-3 mb-3">
+                    <Col>
+                        Connected to account <code>{account}</code>.
+                    </Col>
+                </Row>
+            )}
+            <Form.Group as={Row} className="mb-3">
+                <Col sm={3}>
+                    <Button variant="primary" onClick={handleSubmit} disabled={!connection || isWaiting}>
+                        {isWaiting ? 'Waiting for proof...' : 'Request proof'}
+                    </Button>
+                </Col>
+            </Form.Group>
+            <Row>
+                {error && <Alert variant="danger">{error}</Alert>}
+                {verifiablePresentation && (
+                    <Accordion defaultActiveKey="0">
```

Comment:
There isn't much else on the page so maybe accordion is overkill?

Reply:
It is much nicer than just using `<pre>`. I don't know react-bootstrap well enough to know if there are better alternatives. I think this looks nice for a sample.

Feel free to suggest an alternative from that library for displaying a big JSON blob.